src/components/PoolingWidget/AddFunding.tsx

```diff
@@ -40,7 +40,7 @@ const AddFunding: React.FC<AddFundingProps> = ({ txIdentifier, txReceipt }) => (
     <p>
       <strong>What now?</strong>
       <br />
-      Go to the <Link to="wallet">Wallet page</Link> to deposit at least one of the tokens you selected in your
+      Go to the <Link to="wallet">Balances page</Link> to deposit at least one of the tokens you selected in your
```

Comment:
Why do we sometimes do `to="wallet"` and sometimes `to="/wallet"`, not that it matters

Reply:
Beats me /shrug